mintscript/src/tests/translate/snap.translate.tlock.transfer_00.txt

```diff
@@ -0,0 +1,18 @@
+WitnessScript(
+    Signature(
+        PublicKey(
+            PublicKey {
+                pub_key: Secp256k1Schnorr(
+                    Secp256k1PublicKey {
+                        pubkey_data: PublicKey(
+                            c2b6c1dd66557d1df359d243e8c68ddc2df5cbffea946beb715e7b4218fa43e8f72bed0631a5058760653a922c21685e3cf8020dbc236fa46ad8b86dfa70c7ca,
+                        ),
+                    },
+                ),
+            },
+        ),
+        NoSignature(
+            None,
+        ),
+    ),
+)
```

Comment:
I don't know if this is intended, but all tlock tests exactly match the txn transactions... so timelocks are not translated to scripts... this is probably not intended, right?

Reply:
> I don't know if this is intended, but all tlock tests exactly match the txn transactions... so timelocks are not translated to scripts... this is probably not intended, right?

Tests were mistakenly using the transaction translation modes for timelock-only and reward translation in tests. Fixed now.